zoloto/coords.py

```diff
@@ -79,25 +108,65 @@ class SphericalCoordinates(NamedTuple):
 
     @property
     def rot_x(self) -> float:
-        """Approximate rotation around the x-axis, an alias for ``self.theta``."""
-        return self.theta
+        """
+        Rotation around the x-axis.
+
+        Conventional:  This is unused.
+
+        Legacy: A rotation up to down around the camera, in radians. Values
+                increase as the marker moves towards the bottom of the image.
+                A zero value is halfway up the image.
+        """
+        if os.environ.get('ZOLOTO_LEGACY_AXIS'):
+            return self.phi - (math.pi / 2)
+        else:
+            raise AttributeError("Rotation around this axis is not used")
 
     @property
     def rot_y(self) -> float:
-        """Rotation around the y-axis, an alias for ``self.phi``."""
-        return self.phi
+        """
+        Rotation around the y-axis.
+
+        Conventional: A rotation up to down around the camera, in radians.
+                      Values increase as the marker moves towards the bottom
+                      of the image. A zero value is halfway up the image.
+
+        Legacy: A rotation left to right around the camera, in radians. Values
+                increase as the marker moves towards the right of the image.
+                A zero value is on the centerline of the image.
+        """
+        if os.environ.get('ZOLOTO_LEGACY_AXIS'):
+            return -self.theta
```

Comment:
Issue: A lot of these "new" legacy axis codepaths don't have tests.

That said, I'm slightly concerned the tests aren't failing in their current state...

Reply:
Since this is merging into Peter's fork the tests aren't being run.

Reply:
Ah, that would do it! Do the tests pass locally?